app/Services/Api/ApiPermissionService.php

```diff
@@ -62,6 +40,7 @@ public function saveEntity($request, Entity $entity)
     {
```

Comment:
The function can now type cast the return with `public function saveEntity(...) : CampaignPermission`, and the auto-docs of the function call updated.

Reply:
It doesnt return a CampaignPermission. it returns an array containing all the CampaingPermissions created, if I do the type cast I get an error when trying to post on the api